app/retail/emails.py

```diff
@@ -479,6 +487,7 @@ def render_new_bounty(to_email, bounties, old_bounties, offset=3):
         'subscriber': sub,
         'keywords': ",".join(sub.keywords),
         'email_style': email_style,
+		'email_type': 'new_bounty_notifications'
```

Comment:
all of these email type fields have a mix of tabs/spaces. shuld just be 4 spaces per tab.